src/components/Chips/Chips.jsx

```diff
@@ -19,7 +15,7 @@ const Chips = forwardRef(
     const mergedRef = useMergeRefs({ refs: [ref, componentRef] });
 
     const backgroundColorStyle = useMemo(() => {
-      return { backgroundColor: disabled ? getCSSVar("disabled-background-color") : getSelectedColor(color) };
+      return { backgroundColor: disabled ? getCSSVar("disabled-background-color") : getElementColor(color, true) };
```

Comment:
I made getSelectedColor support 2 palettes - regular and selected palette. for selected palette the function need to recevied second "isSelectedPalette"

Reply:
cool!